portfolio/src/main/java/com/google/sps/data/CommentsResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+import java.util.List;
+
+/**
+ * Class that is used as a JSON Response to the /comments GET endpoint
+ */
+
+public class CommentsResponse {
+    private List<Comment> comments;
+    private String currentUserEmail;
```

Comment:
I think this is better to include as part of the LoginResponse, as it is part of the user authentication information.

Reply:
Moved `currentUserEmail` to `LoginResponse`